src/staging/src/templates/park.js

```diff
@@ -14,7 +14,7 @@ import {
 import { makeStyles } from "@material-ui/core/styles"
 import useScrollSpy from "react-use-scrollspy"
 
-import { capitalizeFirstLetter } from "../utils/helpers";
+import { capitalizeFirstLetter } from "../utils/helpers"
```

Comment:
Are there files missing from this PR? It doesn't seem to be addressing the bug.

Reply:
Oh weird! Yes there should be more than that. Mark I'll reach out if I can't figure it out